tests/storage/hostdisk.go

```diff
@@ -44,3 +50,19 @@ func CreateDiskOnHost(diskPath string) *k8sv1.Pod {
 
 	return pod
 }
+
+func RemoveHostDiskImage(diskPath string, nodeName string) {
+	virtClient := kubevirt.Client()
+	procPath := filepath.Join("/proc/1/root", diskPath)
+
+	virtHandlerPod, err := libnode.GetVirtHandlerPod(virtClient, nodeName)
+	Expect(err).ToNot(HaveOccurred())
```

Comment:
If you are moving the function its best not to add gomega to this file and have this function return error

Reply:
I see that you did that in the last commit, I really dislike adding new commits to change stuff from previous commits.. makes anyone reviewing by commit order waste time on redundant review

Reply:
Well, the new commit is explicit about updating these functions to match conventions. IMO it's better to have one commit that simply moves the functions and, if necessary, other than updates them. I agree we need to avoid spaghetti commits that do things they aren't supposed to (I'm guilty of this), but I still think it's viable to have your commits reflect the development history even if they update earlier changes.

Reply:
When reviewing per-commit like I prefer to do, we will find this issue and comment on it. Then we will find it later as fixed.
This is slowing does the reviewer and the author (which needs to read and respond).

Sometime this is just unavoidable, but sometimes the author can do this fix prior to the move as its own commit (and reasoning). It does not add more work to the reviewer, but it may add a bit more rebase work to the author.

But I also find it fine to always refer to the commit message reasoning and intention. Doing more than it says is a nice-to-have, so reviewers could take that into consideration as well.

There is no right or wrong here.

Regarding the development history: As I see it, the commits should tell a story after the fact, i.e. a non drafted story. The logic should make sense to the reader, not in the sense of how the history of the development has been composed, but how the logic explains best the story.

Reply:
Thanks for the insight @EdDev, yeah, I especially agree with your last point.